cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NewTwoWindingsTransformerConversion.java

```diff
@@ -0,0 +1,426 @@
+        cgmesT2xModel.end1.ratedU = ratedU1;
+        cgmesT2xModel.end1.phaseAngleClock = end1.asInt(STRING_PHASE_ANGLE_CLOCK, 0);
+        cgmesT2xModel.end1.terminal = terminal1;
+
+        if (x1 == 0.0) {
+            cgmesT2xModel.end1.xIsZero = true;
+        } else {
+            cgmesT2xModel.end1.xIsZero = false;
+        }
+
+        cgmesT2xModel.end2.g = end2.asDouble(STRING_G, 0);
+        cgmesT2xModel.end2.b = end2.asDouble(STRING_B);
+        cgmesT2xModel.end2.ratioTapChanger = ratioTapChanger2;
+        cgmesT2xModel.end2.phaseTapChanger = phaseTapChanger2;
+        cgmesT2xModel.end2.ratedU = ratedU2;
+        cgmesT2xModel.end2.phaseAngleClock = end2.asInt(STRING_PHASE_ANGLE_CLOCK, 0);
+        cgmesT2xModel.end2.terminal = terminal2;
+
+        if (x2 == 0.0) {
+            cgmesT2xModel.end2.xIsZero = true;
+        } else {
+            cgmesT2xModel.end2.xIsZero = false;
+        }
+
+        cgmesT2xModel.r = r;
+        cgmesT2xModel.x = x;
+
+        return cgmesT2xModel;
+    }
+
+    private InterpretedT2xModel interpret(CgmesT2xModel cgmesT2xModel, Conversion.Config alternative) {
+
+        AllTapChanger interpretedTapChanger = ratioPhaseAlternative(cgmesT2xModel, alternative);
+        AllShunt interpretedShunt = shuntAlternative(cgmesT2xModel, alternative);
+
+        AllPhaseAngleClock interpretedClock = phaseAngleClockAlternative(cgmesT2xModel, alternative);
+        boolean ratio0AtEnd2 = ratio0Alternative(cgmesT2xModel, alternative);
+
+        InterpretedT2xModel interpretedT2xModel = new InterpretedT2xModel();
+        interpretedT2xModel.r = cgmesT2xModel.r;
+        interpretedT2xModel.x = cgmesT2xModel.x;
+
+        interpretedT2xModel.end1.g = interpretedShunt.g1;
+        interpretedT2xModel.end1.b = interpretedShunt.b1;
+        interpretedT2xModel.end1.ratioTapChanger = interpretedTapChanger.ratioTapChanger1;
+        interpretedT2xModel.end1.phaseTapChanger = interpretedTapChanger.phaseTapChanger1;
+        interpretedT2xModel.end1.ratedU = cgmesT2xModel.end1.ratedU;
+        interpretedT2xModel.end1.terminal = cgmesT2xModel.end1.terminal;
+        interpretedT2xModel.end1.phaseAngleClock = interpretedClock.phaseAngleClock1;
+
+        interpretedT2xModel.end2.g = interpretedShunt.g2;
+        interpretedT2xModel.end2.b = interpretedShunt.b2;
+        interpretedT2xModel.end2.ratioTapChanger = interpretedTapChanger.ratioTapChanger2;
+        interpretedT2xModel.end2.phaseTapChanger = interpretedTapChanger.phaseTapChanger2;
+        interpretedT2xModel.end2.ratedU = cgmesT2xModel.end2.ratedU;
+        interpretedT2xModel.end2.terminal = cgmesT2xModel.end2.terminal;
+        interpretedT2xModel.end2.phaseAngleClock = interpretedClock.phaseAngleClock2;
+
+        interpretedT2xModel.ratio0AtEnd2 = ratio0AtEnd2;
+
+        return interpretedT2xModel;
+    }
+
+    private AllTapChanger ratioPhaseAlternative(CgmesT2xModel cgmesT2xModel, Conversion.Config alternative) {
```

Comment:
For each interpretation, explain what is done (e.g.
```java
// in this case, the ratio and the phase in the CGMES file are considered as side 1
// (cf. Conversion.java for formulas)
// we combine them to follow the IIDM convention
// *insert formula for IIDM convention*
```

Reply:
An initial documentation has been added to the main methods and at class level.